### ISSUE TITLE
[core] NPE when updating Quarkus CLI release to 1.2.0

### ISSUE THREAD

**Issue by gsmet** [jreleaser/jreleaser#930]
Hey there,

I just tried to run a Quarkus CLI release with JReleaser 1.2.0 (I was eager to get the Chocolatey workflow issue fixed) and I get a NPE:

```
[INFO]  Reading configuration
project.website is deprecated since 1.2.0 and will be removed in 2.0.0. Use project.links.homepage instead
project.licenseUrl is deprecated since 1.2.0 and will be removed in 2.0.0. Use project.links.license instead
java.lang.NullPointerException
	at org.jreleaser.engine.context.ModelConfigurer.fillGitProperties(ModelConfigurer.java:159)
	at org.jreleaser.engine.context.ModelConfigurer.autoConfigureGithub(ModelConfigurer.java:91)
	at org.jreleaser.engine.context.ModelConfigurer.configure(ModelConfigurer.java:65)
	at org.jreleaser.engine.context.ContextCreator.create(ContextCreator.java:78)
	at org.jreleaser.cli.AbstractModelCommand.createContext(AbstractModelCommand.java:140)
	at org.jreleaser.cli.AbstractModelCommand.execute(AbstractModelCommand.java:81)
	at org.jreleaser.cli.AbstractCommand.call(AbstractCommand.java:36)
	at org.jreleaser.cli.AbstractModelCommand.call(AbstractModelCommand.java:47)
	at org.jreleaser.cli.AbstractCommand.call(AbstractCommand.java:29)
```

From what I can see, it's an issue with the branch resolution. I can confirm that I set `JRELEASER_BRANCH` to 2.12. So there should be a branch around and apparently, it's not set appropriately.
I have looked for `branch` in the changelog but couldn't find any changes that would require me to adjust `JRELEASER_BRANCH`. And I see it there: https://jreleaser.org/guide/latest/configuration/environment.html#_release .

Our JReleaser setup is located here: https://github.com/quarkusio/quarkus/tree/main/devtools/cli/distribution

**Comment by aalmiray** [comment]
Bug introduced when fixing https://github.com/jreleaser/jreleaser/issues/896 with https://github.com/jreleaser/jreleaser/commit/5a9fe54f5077bada6d53f897b370b3c772c1eee2

**Comment by aalmiray** [comment]
Workaround would be to explicitly set a value for `branch` in the DSL.

**Comment by aalmiray** [comment]
Released in v1.3.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.3.0
